src/lib/y2partitioner/widgets/pages/system.rb

```diff
@@ -172,6 +172,15 @@ def nfs_devices
           device_graph.nfs_mounts
         end
 
+        # @return [Array<Y2Storage::Device>]
```

Comment:
NP: you use `bcaches` in most method names, I find kind of inconsistent to use `bcache_devices` here. Not a big deal, just curious about the rationale (I guess just copy&pasting from `disk_devices`).

Reply:
ok, I will try to unify it to `bcache_devices`